cpp/dolfinx/common/IndexMap.cpp

```diff
@@ -296,12 +296,17 @@ common::stack_index_maps(
   const std::vector<int> out_neighbors(out_neighbor_set.begin(),
                                        out_neighbor_set.end());
 
+  // NOTE: create uniform weights as a workaround to issue
+  // https://github.com/pmodels/mpich/issues/5764
+  const std::vector<int> in_weights(in_neighbors.size(), 1);
+  const std::vector<int> out_weights(out_neighbors.size(), 1);
+
```

Comment:
It's not really needed here, as we soon free the comm anyway, and it never gets copied.

Reply:
It can be duplicated (and probably will be) by functions that access the IndexMap comms.